mailpoet/tests/docker/docker-compose.yml

```diff
@@ -71,7 +71,7 @@ services:
       - mailhog-data:/mailhog-data
 
   wordpress:
-    image: mailpoet/wordpress:${WORDPRESS_IMAGE_VERSION:-wp-6.4_php8.0_20231114.1}
```

Comment:
I think we don't want to change the used PHP version. The previous image for WP 6.4 was with PHP 8.0, and the new one is with PHP 8.1. Let me know if you need some assistance here.

Reply:
Yes, but as of WP update to 6.5, we had to bump also PHP version. @JanJakes has probably more information to this.